package.json

```diff
@@ -51,6 +52,7 @@
     "@graphql-codegen/typed-document-node": "^2.3.10",
     "@graphql-codegen/typescript": "^2.8.5",
     "@graphql-codegen/typescript-operations": "^2.5.10",
-    "graphql-language-service": "^5.0.6"
+    "graphql-language-service": "^5.0.6",
+    "node-fetch": "^2.0.0"
```

Comment:
Took an older version of node-fetch due to the issues with the ESM enforcement and how TS resolves it 😅